backend/modules/pdf_processing/repository.py

```
@@ -0,0 +1,352 @@
+                        primary_goals=figure.get("primary_goals", []) or figure.get("primaryGoals", []),
+                        personality_traits=traits,
+                        image_url=figure.get("image_url") or figure.get("imageUrl"),
+                        created_at=datetime.utcnow(),
+                        updated_at=datetime.utcnow()
+                    )
+                    self.db.add(persona)
+                    existing_persona_names.add(persona_name)
+            
+            self.db.commit()
+            debug_log(f"[REPOSITORY] Successfully saved autofill data for scenario {scenario_id}")
+            return True
```

Ruff (0.14.5)

Consider moving this statement to an `else` block

(TRY300)